content/applications/hr/attendances.rst

```diff
@@ -1,19 +1,474 @@
+
+- :guilabel:`Search...`: Tap on the :guilabel:`Search...` field and enter the person's name. As the
+  name is typed in, the matching results are displayed on the screen.
+- :guilabel:`Department`: To quickly filter the presented employees, tap on a department to display
+  only those employees that are part of that department. The :guilabel:`Departments` are listed on
+  the left side of the screen, and the number at the end of each department indicates how many
+  employees are part of the department, and will be displayed when selected.
+
+PIN number
+~~~~~~~~~~
+
+If the :guilabel:`Employee PIN Identification` option was activated in the :ref:`kiosk settings
+<attendances/kiosk-settings>` section of the configuration menu, the employee is prompted to enter a
+PIN number when manually checking in or out.
+
+After the employee is selected, a number pad appears with a message. When checking in,
+:guilabel:`(Employee) Welcome! Please enter your PIN to check in` appears above the numbers. When
+checking out, :guilabel:`(Employee) Want to check out? Please enter your PIN to check out` appears
+above the numbers. Tap in the PIN number using the number pad, then tap :guilabel:`OK` when done.
+The employee is then checked in or out, and a :ref:`confirmation message
+<attendances/confirmation>` appears.
+
+.. image:: attendances/enter-pin.png
+   :align: center
+   :alt: The pop-up that appears when prompted to enter a pin number.
+
+.. _attendances/confirmation:
+
+Confirmation message
+--------------------
+
+When an employee checks in or out, a confirmation message appears with all the check in or check out
+information. When checking in, a welcome message appears, as well as the date and time of check in.
+An :guilabel:`Hours Previously Today: XX:XX` field also appears, displaying any time already logged
+for that employee for the day. If no time has already been logged, the value displayed is `00:00`.
+Beneath the message is an :guilabel:`OK` button. To exit the screen before the preset time in the
+kiosk, tap the :guilabel:`OK` button.
+
+When checking out, the message displays a goodbye message, with the date and time of check out, and
+the total hours logged for the day. Beneath the message is a :guilabel:`Goodbye` button. To exit the
+screen before the preset time, tap the :guilabel:`Goodbye` button.
+
+.. image:: attendances/goodbye-message.png
+   :align: center
+   :alt: The goodbye message with all the employee's check out information.
+
+Overview
+========
+
+When entering the *Attendances* application, the :guilabel:`Overview` dashboard is presented,
+containing all the check in and check out information for the signed in user. If the user has
+specific :ref:`access rights <attendances/access-rights>` and/or are :ref:`approvers
+<attendances/approvers>` for specific employees, then those additional employee's check in and check
+out information is also visible on the :guilabel:`overview` dashboard.
```

Comment:
```suggestion
out information is also visible on the :guilabel:`Overview` dashboard.
```

Reply:
I wrote this with a capital O because on the menu at the top, the button to get to the Overview is with an "O", and I have it in a gui since I was sort of referencing the button, even though I didn't explicitly state that.